contents/blog/data-warehouse-launch.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2024-06-08
+title: You can now sync data from external sources into PostHog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/data_warehouse_92b43aa9de.jpg
+author:
+  - joe-martin
+featuredImageType: full
+tags:
+  - PostHog news
+---
+
+## Tl;dr
+
+- Data warehouse is now out of beta, with pre-built connectors for Stripe, Hubspot, Zendesk, and more
+- You can use synced data across any PostHog tool, including analyzing data with insights and HogQL 
```

Comment:
not fully true yet with feature flags/cohorts/experiments

Reply:
```suggestion
- You can use synced data across most PostHog tools, such as analyzing data with insights and HogQL 
```